### PR TITLE
Clean application boot

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Listener/ConfigureCastorListener.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{
+    public function __construct(
+        private readonly ErrorHandler $errorHandler,
+        private readonly OutputInterface $output,
+        private readonly LoggerInterface $logger = new NullLogger(),
+    ) {
+    }
+
+    #[AsEventListener()]
+    public function generateStubs(BeforeApplicationInitializationEvent $event): void
```
Comment (pyrech) [inline src/Listener/ConfigureCastorListener.php:25]: The method is not right :eyes:

**Review by pyrech — COMMENTED** [review]
Just a small nitpick, but looks good otherwise

**Review by pyrech — APPROVED** [review]
Lgtm. Could you add some documentation about the new event?

**Comment by lyrixx** [comment]
> Lgtm. Could you add some documentation about the new event?

For now, I prefer to keep it internal. I don't see legitimate use case. Hope it's okay for you. I

**Comment by pyrech** [comment]
Could you mark the event class internal then?

**Comment by lyrixx** [comment]
> Could you mark the event class internal then?

Oupsy, fixed 👍🏼

**Review by pyrech — APPROVED** [review]
Thanks

**Pull request merged**